tests/ui/traits/non_lifetime_binders/type-match-with-late-bound.stderr

```diff
@@ -7,5 +7,20 @@ LL | #![feature(non_lifetime_binders)]
    = note: see issue #108185 <https://github.com/rust-lang/rust/issues/108185> for more information
    = note: `#[warn(incomplete_features)]` on by default
 
-warning: 1 warning emitted
+error[E0309]: the placeholder type `!1_"F"` may not live long enough
```

Comment:
I don't really care about fixing this diagnostic regression for now, since it's behind a super unstable feature gate.